debugging-contd/problem-set-debugging-contd.md

```diff
@@ -1,125 +1,88 @@
 # Problem Set: Debugging Continued
 
-<iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Embed.aspx?pid=465884e5-a350-4020-abb8-ad9101852567&autoplay=false&offerviewer=true&showtitle=true&showbrand=false&captions=true&interactivity=all" height="405" width="720" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe>
+## Overview
 
-Moira has been working on some code related to [Conway's Game of Life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life), but has run into some bugs. Today's task is to fork Moira's Replit and help Moira debug her code.
+Dahlia is a geneticist and programmer who is stuck while debugging her program. Dahlia has been tasked with analyzing genetic sequences which are provided as strings of characters. 
 
-Before getting started, here is some important information about the project that Moira is working on:
+Our objective is to fork [Dahlia's Replit](https://replit.com/@adacore/EncodeAndDecodeGeneticSequencesStarter) and help Dahlia debug her code.
 
-Conway's Game of Life is a classic [cellular automata](https://en.wikipedia.org/wiki/Cellular_automaton) simulation in computer science. In Conway's Game of Life, there is a stage or grid, and each point in the stage is a cell. Each generation, every cell on the stage lives or dies based on a few simple rules.
+She needs to determine if each sequence is DNA or RNA, and compress the sequence for storage. Dahlia will compress them by replacing repeating characters in the string with a single character followed by a number representing how many times the character was repeated. 
 
-1. Any live cell with < 2 neighbors dies
-1. Any live cell with 2-3 neighbors lives
-1. Any live cell with > 3 neighbors dies
-1. Any dead cell with 3 neighbors comes alive
+Dahlia's code can either be ran in test mode or production mode. When you run the project on Replit, the console will prompt you to enter `t` to test the project, or `r` to run the code in production mode.
 
-A cell's neighbors are the 8 cells immediately around the cell.
+Before jumping into debugging, please take a few minutes to read through the `PROBLEM_SET_README.md` file in the Replit for further information on how Dahlia is trying to achieve her goals and what the files in her project should be doing. 
 
-Before going further, take a minute to play with this [Game of Life simulator](https://playgameoflife.com/).  Add a few points to the grid and press start to watch the simulation.
+When you're ready, come back to this page to answer the questions below about the debugging process and the bugs you found.
```

Comment:
Since students are meant to manually debug the bugs in the replit, what do you think about adding a line about refraining from using ChatGPT for now?

"The focus of this exercise is to practice identifying bugs. We will continue to practice this skill during our time together at Ada and we should feel comfortable doing so without the use of AI tools like ChatGPT. Later, we'll practice debugging code with the assistance of prompting. For now, we'll stick to tools like rubber ducking or performing web searches to fix these bugs."